src/main/steps/c100-rebuild/screening-questions/alternative-resolution/going-to-court/template.njk

```diff
@@ -0,0 +1,25 @@
+{% extends "common/template.njk" %}
+{% block page_content %}
+<h1 class="govuk-heading-xl">{{title}}</h1>
+<p class="govuk-body">{{titleDetail}}</p>
+<div class="govuk-inset-text"> {{insetText}}</div>
+<h1 class="govuk-heading-l">{{secondTitle}}</h1>
+<p class="govuk-body">{{para1}}</p>
```

Comment:
This can be improved by defining an Array of statements in content.ts & add a loop to iterate & display here.